### ISSUE TITLE
Can't extract audio in wav format from youtube video 

### ISSUE THREAD

**Issue by jcn199729** [yt-dlp/yt-dlp#1704]
### Checklist

- [X] I'm asking a question and not reporting a bug/feature request
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar questions including closed ones

### Question

![image](https://user-images.githubusercontent.com/58527362/142356535-9f255427-0fb1-4a2d-95ce-d7713b633a3c.png)

**Comment by nicolaasjan** [comment]
Same issue as https://github.com/yt-dlp/yt-dlp/issues/1645.
Is fixed with https://github.com/yt-dlp/yt-dlp/commit/39c04074e7e108bc6e36f3a34ef08a163663144a.

Just wait for the new release or build yt-dlp from the `master` branch.
As you can see, it then works:

```shell
dlp -v --ignore-config -x --audio-format wav pmQhKRKnhZY
[debug] Command-line config: ['-v', '--ignore-config', '-x', '--audio-format', 'wav', 'pmQhKRKnhZY']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, err UTF-8, pref UTF-8
[debug] yt-dlp version 2021.11.16 (zip)
[debug] Python version 3.6.9 (CPython 64bit) - Linux-5.4.0-90-generic-x86_64-with-LinuxMint-19.3-tricia
[debug] exe versions: ffmpeg N-104554-gdb932241ee-Nico-20211116-With-VP9-Timestamp-Fix (setts,fdk), ffprobe N-104554-gdb932241ee-Nico-20211116-With-VP9-Timestamp-Fix, phantomjs 2.1.1
[debug] Optional libraries: Cryptodome, keyring, mutagen, sqlite, websockets
[debug] Proxy map: {}
[debug] [youtube] Extracting URL: pmQhKRKnhZY
[youtube] pmQhKRKnhZY: Downloading webpage
[youtube] pmQhKRKnhZY: Downloading android player API JSON
[youtube] pmQhKRKnhZY: Downloading player 68e11abe
[debug] Saving youtube-nsig.68e11abe to cache
[debug] [youtube] Decrypted nsig DJrhd6OjDy5PQbn => EH2o8mTB4T6kWA
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, codec:vp9.2, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), acodec, lang, proto, filesize, fs_approx, tbr, vbr, abr, asr, vext, aext, hasaud, id
[info] pmQhKRKnhZY: Downloading 1 format(s): 251
[debug] Invoking downloader on "https://r5---sn-5hnekn7d.googlevideo.com/videoplayback?expire=1637244584&ei=SAqWYeqjKtqa1gLk9KiwBg&ip=<REDACTED>&id=o-AMCXFx0EJv0nkLNShhRVt4-o5EDWQFiK6VVIdsf5YPLK&itag=251&source=youtube&requiressl=yes&mh=Ne&mm=31%2C29&mn=sn-5hnekn7d%2Csn-5hne6nsz&ms=au%2Crdu&mv=m&mvi=5&pl=19&initcwndbps=2095000&vprv=1&mime=audio%2Fwebm&gir=yes&clen=13012127&dur=794.081&lmt=1637004447323446&mt=1637222466&fvip=5&keepalive=yes&fexp=24001373%2C24007246&c=ANDROID&txp=4432434&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cvprv%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRQIgS4oqHjPkQ1CA6afO2BvfO3JbhzzyDaLORZIjBbqLQbICIQDuk6FizBA57kx6anyuD4cMyZFOo4zJA5Y0wZzPNM8hQA%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIhAPk_f9dGUeMVqvg49xLm65x_JwtvcGOcg474ykyKMvHWAiAfN2Kz7wsXob96dzrm5WXSImvfn8wrnQqbyoYiy0WCDA%3D%3D"
[download] Destination: 叒一个划时代的产品？_拿火智能吉他 LAVA ME 3【值不值得买第519期】 [pmQhKRKnhZY].webm
[download] 100% of 12.41MiB in 00:01                 
[debug] ffmpeg command line: ffprobe -show_streams 'file:叒一个划时代的产品？_拿火智能吉他 LAVA ME 3【值不值得买第519期】 [pmQhKRKnhZY].webm'
[ExtractAudio] Destination: 叒一个划时代的产品？_拿火智能吉他 LAVA ME 3【值不值得买第519期】 [pmQhKRKnhZY].wav
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i 'file:叒一个划时代的产品？_拿火智能吉他 LAVA ME 3【值不值得买第519期】 [pmQhKRKnhZY].webm' -vn -f wav 'file:叒一个划时代的产品？_拿火智能吉他 LAVA ME 3【值不值得买第519期】 [pmQhKRKnhZY].wav'
Deleting original file 叒一个划时代的产品？_拿火智能吉他 LAVA ME 3【值不值得买第519期】 [pmQhKRKnhZY].webm (pass -k to keep)

```

**Comment by jcn199729** [comment]
> Same issue as #1645. Is fixed with [39c0407](https://github.com/yt-dlp/yt-dlp/commit/39c04074e7e108bc6e36f3a34ef08a163663144a).
> 
> Just wait for the new release or build yt-dlp from the `master` branch. As you can see, it then works:
> 
> ```shell
> dlp -v --ignore-config -x --audio-format wav pmQhKRKnhZY
> [debug] Command-line config: ['-v', '--ignore-config', '-x', '--audio-format', 'wav', 'pmQhKRKnhZY']
> [debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, err UTF-8, pref UTF-8
> [debug] yt-dlp version 2021.11.16 (zip)
> [debug] Python version 3.6.9 (CPython 64bit) - Linux-5.4.0-90-generic-x86_64-with-LinuxMint-19.3-tricia
> [debug] exe versions: ffmpeg N-104554-gdb932241ee-Nico-20211116-With-VP9-Timestamp-Fix (setts,fdk), ffprobe N-104554-gdb932241ee-Nico-20211116-With-VP9-Timestamp-Fix, phantomjs 2.1.1
> [debug] Optional libraries: Cryptodome, keyring, mutagen, sqlite, websockets
> [debug] Proxy map: {}
> [debug] [youtube] Extracting URL: pmQhKRKnhZY
> [youtube] pmQhKRKnhZY: Downloading webpage
> [youtube] pmQhKRKnhZY: Downloading android player API JSON
> [youtube] pmQhKRKnhZY: Downloading player 68e11abe
> [debug] Saving youtube-nsig.68e11abe to cache
> [debug] [youtube] Decrypted nsig DJrhd6OjDy5PQbn => EH2o8mTB4T6kWA
> [debug] Sort order given by extractor: quality, res, fps, hdr:12, source, codec:vp9.2, lang, proto
> [debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), acodec, lang, proto, filesize, fs_approx, tbr, vbr, abr, asr, vext, aext, hasaud, id
> [info] pmQhKRKnhZY: Downloading 1 format(s): 251
> [debug] Invoking downloader on "https://r5---sn-5hnekn7d.googlevideo.com/videoplayback?expire=1637244584&ei=SAqWYeqjKtqa1gLk9KiwBg&ip=<REDACTED>&id=o-AMCXFx0EJv0nkLNShhRVt4-o5EDWQFiK6VVIdsf5YPLK&itag=251&source=youtube&requiressl=yes&mh=Ne&mm=31%2C29&mn=sn-5hnekn7d%2Csn-5hne6nsz&ms=au%2Crdu&mv=m&mvi=5&pl=19&initcwndbps=2095000&vprv=1&mime=audio%2Fwebm&gir=yes&clen=13012127&dur=794.081&lmt=1637004447323446&mt=1637222466&fvip=5&keepalive=yes&fexp=24001373%2C24007246&c=ANDROID&txp=4432434&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cvprv%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AOq0QJ8wRQIgS4oqHjPkQ1CA6afO2BvfO3JbhzzyDaLORZIjBbqLQbICIQDuk6FizBA57kx6anyuD4cMyZFOo4zJA5Y0wZzPNM8hQA%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AG3C_xAwRQIhAPk_f9dGUeMVqvg49xLm65x_JwtvcGOcg474ykyKMvHWAiAfN2Kz7wsXob96dzrm5WXSImvfn8wrnQqbyoYiy0WCDA%3D%3D"
> [download] Destination: 叒一个划时代的产品？_拿火智能吉他 LAVA ME 3【值不值得买第519期】 [pmQhKRKnhZY].webm
> [download] 100% of 12.41MiB in 00:01                 
> [debug] ffmpeg command line: ffprobe -show_streams 'file:叒一个划时代的产品？_拿火智能吉他 LAVA ME 3【值不值得买第519期】 [pmQhKRKnhZY].webm'
> [ExtractAudio] Destination: 叒一个划时代的产品？_拿火智能吉他 LAVA ME 3【值不值得买第519期】 [pmQhKRKnhZY].wav
> [debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i 'file:叒一个划时代的产品？_拿火智能吉他 LAVA ME 3【值不值得买第519期】 [pmQhKRKnhZY].webm' -vn -f wav 'file:叒一个划时代的产品？_拿火智能吉他 LAVA ME 3【值不值得买第519期】 [pmQhKRKnhZY].wav'
> Deleting original file 叒一个划时代的产品？_拿火智能吉他 LAVA ME 3【值不值得买第519期】 [pmQhKRKnhZY].webm (pass -k to keep)
> ```

Greatly thanks :)
